src/main/java/sa/assertj/iterables/Are.java

```diff
@@ -0,0 +1,34 @@
+package sa.assertj.iterables;
+
+import org.assertj.core.api.Condition;
+import sa.assertj.Experiment;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Are extends Experiment{
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int i = 0; i < numOfSamples; i++) {
+            ArrayList<Integer> list = new ArrayList<>();
+            Random rand = new Random();
+            int element = rand.nextInt();
+            for (int j = 0; j < size; j++) {
+                list.add(element);
+            }
+
+            Condition<Integer> condition = new Condition<>(s -> s.equals(element), "equals");
```

Comment:
```suggestion
            Condition<Integer> condition = new Condition<>(s -> s.equals(element), "equal to " + element);
```

This also goes for the other conditions in the other classes, cause it should finish the sentence "All of these are _."

Although feel like maybe we could switch it up to be something else? Like "positive"? To generate that, we can use `rand.nextInt(Integer.MAX_VALUE)` since this bounded one makes a uniform distribution from 0 inclusive to the input value exclusive.